x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/rule.tsx

```diff
@@ -71,6 +71,9 @@ export function RuleComponent({
 }: RuleComponentProps) {
   const { ruleTypeRegistry, actionTypeRegistry, alertsTableConfigurationRegistry } =
     useKibana().services;
+  // The lastReloadRequestTime should be updated when the refreshToken changes
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const lastReloadRequestTime = useMemo(() => +new Date(), [refreshToken]);
```

Comment:
nit: For readability maybe we can do `new Date().getTime()`. `+new Date()` is a bit difficult to understand what it does.

Reply:
This isn't exactly what memo is meant to be used for. Is this a hack or is there an underlaying issue? Wouldn't this be done with an effect?

Reply:
Changed to `getTime()` in cae83a76c9c02835b17f6470838788764bf35cef ✅ 

@jcger it's a temporary solution based on the two tools we already have at our disposal (`refreshToken` for the rule page and `lastReloadRequestTime` for the alerts table). In `alerts-table-refactor` I introduced an imperative handle to forward a couple of APIs such as the original `refresh` function, which we could use as a more definitive and cleaner solution 🙂